docs/ml/_index.md

```diff
@@ -15,7 +15,7 @@ menuindent: true
 
 {{<imgproc src="/ml/training.png" class="alignright" resize="400x" declaredimensions=true alt="ML training">}}
 
-Viam includes a built-in machine learning (ML) model service which provides your machine with the ability to learn from data and adjust its behavior based on insights gathered from that data.
+Viam includes a built-in [machine learning (ML) model service](//ml/deploy/) which provides your machine with the ability to learn from data and adjust its behavior based on insights gathered from that data.
```

Comment:
So with the model frameworks stuff i've been working on it looks like there will be more services like `onnx_cpu`, and there are already the modular services. Do you think it might be good to rephrase with that in mind as built-in makes it seem like just one?

Or is built-in not referring to `tflite_cpu` specifically but the type of service and would that still be accurate as the api type is built in?

Reply:
Yes - good point
```suggestion
Machine Learning (ML) provides your machines with the ability to adjust its behavior based on models that recognize patterns or make predictions. 
```

Reply:
That works with the sentence below